### DISCUSSION TITLE
Could Slint possibly support more image formats, such as GIF ,BMP and others?

### DISCUSSION THREAD

**Discussion by ngxtp** [slint-ui/slint#7218] [General] [answered]
I noticed that only the PNG and JPEG formats are enabled for images libraries .

**Comment by ogoffart** [comment, marked as answer]
In fact, Slint can support all format that are supported by the `image` crate.
It is just that the features are disabled by default.

But you can add that in your Cargo.toml:
```toml
image = { version = "0.24", features = ["default"] }
```

The version must be the same as the one used by Slint

**Reply by ApolloRoboto** [reply]
I've added this exact dependency and made sure the gif feature was included, but gifs doesn't seems to be animated

**Reply by ogoffart** [reply]
Animated gif is not yet implemented.
Tracked in https://github.com/slint-ui/slint/issues/2081

**Reply by ApolloRoboto** [reply]
Thank you!
